internal/test/e2e/e2e_test.go

```diff
@@ -153,7 +153,7 @@ func buildOperator(t *testing.T, ctx context.Context) (imageTar string, err erro
 	operatorImage := "etcd-cluster-operator:test"
 
 	// Build the operator.
-	out, err := exec.CommandContext(ctx, "docker", "build", "-t", operatorImage, *fRepoRoot).CombinedOutput()
+	out, err := exec.CommandContext(ctx, "docker", "build", "-t", operatorImage, *fRepoRoot, "--build-arg=debug=true").CombinedOutput()
```

Comment:
I did `make kind` on my laptop
I *did* get an alpine based `manager` container, but when I exec into it I don't see these utilities.
Does a build arg necessesarily get exported as an environment variable?

Reply:
should be fixed - i had to define the build `ARG` after the `FROM` for some reason